Command/RemoveThumbsCommand.php

```diff
@@ -61,27 +64,22 @@ public function configure()
     /**
      * {@inheritdoc}
      */
-    public function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Because the parent class method is `protected`. So I don't see any reasons to keep this `public`.